webhook-server-container/github_api.py

```diff
@@ -361,43 +365,51 @@ def add_size_label(self, pull_request, current_size_label=None):
                 self._remove_label(pull_request=pull_request, label=current_size_label)
                 self._add_label(pull_request=pull_request, label=label)
 
-    def label_by_user_comment(self, pull_request, user_request, reviewed_user):
-        _label = user_request[1]
+    def label_by_user_comment(self, pull_request, user_request, remove, reviewed_user):
+        if not any(
+            user_request.lower().startswith(label_name)
+            for label_name in USER_LABELS_DICT
+        ):
+            self.app.logger.info(
+                f"Label {user_request} is not a predefined one, will not be added / removed."
+            )
+            return
 
         # Skip sonar tests comments
-        if "sonarsource.github.io" in _label:
+        if "sonarsource.github.io" in user_request:
             return
 
         if not any(
```

Comment:
dup of https://github.com/myakove/github-gitlab-webhook-server/pull/81/files#diff-18dd92ec071a4f9657ab44d2874deb63341bac5d1efdf963e4751d733ff64543R369 no?

Reply:
I don't understand where is the dup?